### ISSUE TITLE
[bug] Beacons are Silently Failing to Populate in DB

### ISSUE THREAD

**Issue by jabbate19** [spellshift/realm#805]
**Describe the bug**
I am running agents and tavern and seeing the logs to ClaimTask. Yet, there are no beacons being registered in the DB or Web GUI.

**To Reproduce**

I am running Tavern in Openshift/OKD and Imix on a variety of hosts:

- Ubuntu 22.04
- Debian 12
- Rocky 8.6
- Fedora 41
- PfSense 2.7.0
- Windows 10
- Windows Server 2022

**Expected behavior**
Agents are registered in DB and appear in WebGUI

**Screenshots**
If applicable, add screenshots to help explain your problem

<img width="716" alt="image" src="https://github.com/user-attachments/assets/fecc5e07-3b59-4863-a65b-f5f5b6211a2f">
<img width="309" alt="image" src="https://github.com/user-attachments/assets/8916714b-4ce7-4246-aae2-d3755bf06ad4">

**Additional context**
I'll try and debug this during the weekend

**Comment by hulto** [comment]
Do you have network capture or logs between server and DB?

**Comment by jabbate19** [comment]
No, but there is definitely connectivity. It is able to add my user (shown in photo above)

**Comment by jabbate19** [comment]
I might make a PR just to add more logging to tavern. Will make solving issues like this much easier.

**Comment by hulto** [comment]
Do you have the debug log from the agent?

**Comment by jabbate19** [comment]
This was user error
